Bounding_volumes/include/CGAL/Min_annulus_d.h

```diff
@@ -272,7 +269,7 @@ class Min_annulus_d {
 
   typedef  CGAL::Join_input_iterator_1<
     Basic_variable_index_iterator,
-    CGAL::Unary_compose_1<Point_by_index,Divide> >
+    std::function<Point(int)> >
```

Comment:
Here I understand the need for `std::function`: you need a concrete type name. Do no forget to include `<funtional>`.